base/operators.jl

```diff
@@ -1188,6 +1239,19 @@ used to implement specialized methods.
 """
 <(x) = Fix2(<, x)
 
+"""
+    ≮(x)
+
+Create a function that compares its argument to `x` using [`≮`](@ref), i.e.
+a function equivalent to `y -> y ≮ x`.
+The returned function is of type `Base.Fix2{typeof(≮)}`, which can be
+used to implement specialized methods.
+
+!!! compat "Julia 1.2"
```

Comment:
Seems like the wrong version

Reply:
@stevengj, Oh, I see, I thought this note was related to `Fix2`, but how do I know what version of Julia will integrate these modifications? What version should I specify?

Reply:
I specified 1.8, and added some docstrings for inequality operators added.